internal/test/factory.go

```diff
@@ -69,6 +69,14 @@ func GenerateBlobTxs(numTxs, blobsPerPfb, blobSize int) [][]byte {
 	return txs
 }
 
+func GenerateBlobs(blobSizes ...int) []*blob.Blob {
```

Comment:
should we give this a godoc

Reply:
It's in an `internal` director, so this wouldn't be public to users and I think it should be self explanatory enough to us core developers so forgive me for letting this one pass